static/css/todo.css

```diff
@@ -163,19 +178,33 @@
 }
 
 /* Responsive design for smaller screens */
-@media (max-width: 768px) {
+@media (max-width: 48rem) {
```

Comment:
max-width should stay in pixels, sorry should have clarified! rem is relative to the screen size and scales. Using px here allows the change to be consistent across screens